static/app/components/metrics/queryBuilder.tsx

```diff
@@ -345,14 +353,17 @@ export const QueryBuilder = memo(function QueryBuilder({
           >
             {selectedMeta && isVirtualMetric(selectedMeta) ? (
               <QueryFieldGroup>
-                <Label>{t('Where')}</Label>
+                <QueryFieldGroup.Label css={labelCss({theme})}>
```

Comment:
I wonder if calling this creates a new style sheet & `className` every render 🤔 
`labelCss({theme})`

Reply:
I will improve it in a follow-up